parm/post/upp.yaml

```diff
@@ -18,13 +18,19 @@ analysis:
     rdaod: True
   data_in:
     copy:
+<<<<<<< HEAD
+      - ["{{ PARMgfs }}/post/postxconfig-NT-GFS-ANL.txt", "{{ DATA }}/postxconfig-NT.txt"]
+      - ["{{ COMIN_ATMOS_ANALYSIS }}/{{ RUN }}.t{{ current_cycle | strftime('%H') }}z.atmanl.nc", "{{ DATA }}/{{ atmos_filename }}"]
+      - ["{{ COMIN_ATMOS_ANALYSIS }}/{{ RUN }}.t{{ current_cycle | strftime('%H') }}z.sfcanl.nc", "{{ DATA }}/{{ flux_filename }}"]
+=======
       - ["{{ PARMgfs }}/post/gfs/postxconfig-NT-gfs-anl.txt", "{{ DATA }}/postxconfig-NT.txt"]
       - ["{{ COM_ATMOS_ANALYSIS }}/{{ RUN }}.t{{ current_cycle | strftime('%H') }}z.atmanl.nc", "{{ DATA }}/{{ atmos_filename }}"]
       - ["{{ COM_ATMOS_ANALYSIS }}/{{ RUN }}.t{{ current_cycle | strftime('%H') }}z.sfcanl.nc", "{{ DATA }}/{{ flux_filename }}"]
+>>>>>>> 0953c0f38efb5fbe425520dc40b24dcfdd33a98f
```

Comment:
You need to combine these two sets of changes and remove the conflict markers.

Reply:
Yes, I combined the changes, then removed conflict markers, finally git add/commit/push

Reply:
please resolve teh conflict markers

Reply:
@WalterKolczynski-NOAA 
I have combined these two sets of changes and removed the conflict markers as follows:
 
 21       - ["{{ PARMgfs }}/post/gfs/postxconfig-NT-gfs-anl.txt", "{{ DATA }}/postxconfig-NT.txt"]
 22       - ["{{ COMIN_ATMOS_ANALYSIS }}/{{ RUN }}.t{{ current_cycle | strftime('%H') }}z.atmanl.nc", "{{ DATA }}/{{ atmos_filename }}"]
 23       - ["{{ COMIN_ATMOS_ANALYSIS }}/{{ RUN }}.t{{ current_cycle | strftime('%H') }}z.sfcanl.nc", "{{ DATA }}/{{ flux_filename }}"]

Then did git add upp.yaml,  git commit -m "update upp..yaml" and git push origin feature/atmos_upp_com_in_out
Could you tell me  why the conflict is still there. 
Thanks.

Reply:
It's been resolved now.